samples/browser/pub_sub/index.ts

```diff
@@ -211,10 +212,13 @@ async function main() {
         })
         .then((subscription) => {
             log(`start publish`)
-            connection.publish(subscription.topic, 'NOTICE ME', subscription.qos);
+            var msg_count = 0;
+            connection.publish(subscription.topic, `NOTICE ME {${msg_count}}`, subscription.qos);
```

Comment:
technically msg_count is always 0 here

Reply:
The msg_count will be update in the setInterval callback.